storage/kvstore/filesystemstore/tests/UNITTESTS/FileSystemStore/CMakeLists.txt

```diff
@@ -0,0 +1,48 @@
+# Copyright (c) 2021 ARM Limited. All rights reserved.
+# SPDX-License-Identifier: Apache-2.0
+
+set(TEST_NAME filesystemstore-unittest)
+
+add_executable(${TEST_NAME})
+
+target_include_directories(${TEST_NAME}
+    PRIVATE
+        ${MBED_PATH}
+)
+
+target_compile_definitions(${TEST_NAME} 
+    PRIVATE
+        UNITTEST
+        MBED_LFS_READ_SIZE=64
+        MBED_LFS_PROG_SIZE=64
+        MBED_LFS_BLOCK_SIZE=512
+        MBED_LFS_LOOKAHEAD=512
```

Comment:
We could remove this config stuff if we link against a littlefs library target instead.

Reply:
Can we address this issue as part of "Refactor components for independent unit testing"?